grpc/nodejs/BUILD

```diff
@@ -71,24 +71,22 @@ genrule(
     #    Output typescript to (you guessed it) the genrule output directory
     #    Set the .proto file relative path to the WORKSPACE this command is being run from
     #    Use the .proto files found in the :proto-raw-buffers filegroup as inputs.
-    cmd = "ln -s $(execpath //grpc/nodejs:protoc-gen-ts).runfiles/npm/node_modules ./node_modules ;\
-        $(execpath //grpc/nodejs:grpc_tools_node_protoc) \
+    cmd = "$(execpath //grpc/nodejs:grpc_tools_node_protoc) \
             --plugin='protoc-gen-ts=$(rootpath @npm//:node_modules/grpc_tools_node_protoc_ts/bin/protoc-gen-ts)' \
             --js_out='import_style=commonjs,binary:./$(@D)/' \
             --grpc_out='grpc_js:./$(@D)/' \
             --ts_out='grpc_js:./$(@D)/' \
-            --proto_path=`dirname $(execpath //:WORKSPACE)` \
-            $(execpaths //protobuf:proto-raw-buffers);",
-    # Dependencies for the above command. the //:WORKSPACE label is required to locate the relative path for the .proto files so they can import each other.
+            --proto_path=. \
+            $(locations //protobuf:proto-raw-buffers);",
```

Comment:
locations is recommended against in the documentation- as it resolves to either rootpaths or execpaths, we should probably explicitly say either rootpaths or execpaths as appropriate.

Reply:
@Phillammon addressed in https://github.com/graknlabs/protocol/pull/93/commits/97ce438beac7193c10fd7248ef76820ed1b9a59f